docs/source/reference/rest.md

```diff
@@ -55,8 +60,12 @@ generating an API token is available from the JupyterHub user interface:
 
 ![Request API token page](../images/token-request.png)
 
+<p style="text-align: center">Figure 1.0: JupyterHub's Request API token page</p>
```

Comment:
inline html probably isn't the right way to add a caption. Let's try using the builtin [figure directive](https://jupyterbook.org/en/stable/content/figures.html#figures).